pocs/focuser/birger.py

```diff
@@ -112,7 +130,6 @@ def move_to(self, position):
         Does not do any checking of the requested position but will warn if the lens reports hitting a stop.
         Returns the actual position moved to in lens encoder units.
         """
-        self.logger.debug('Moving focus to {} encoder units'.format(position))
```

Comment:
So, to actually answer the question, I don't think there would be a lot logged from the focuser at the `INFO` level but that's fine. Basically a message for creation. Maybe before and after focusing if that whole procedure takes time.